packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -166,18 +166,15 @@ export const DangerZone = ({
     unverify({ variables: { email: email.email } });
   };
 
-  const [disableAccount, { loading: disableLoading }] = useMutation(
-    DISABLE_ACCOUNT,
-    {
-      onCompleted: () => {
-        window.alert('The account has been disabled.');
-        onCleared();
-      },
-      onError: () => {
-        window.alert('Error disabling account');
-      },
-    }
-  );
+  const [disableAccount] = useMutation(DISABLE_ACCOUNT, {
+    onCompleted: () => {
+      window.alert('The account has been disabled.');
+      onCleared();
+    },
+    onError: () => {
+      window.alert('Error disabling account');
+    },
+  });
```

Comment:
We should include a test for this in the follow-up test ticket....

Reply:
Ah yeah, good call. Wasn't really my change. It looks like prettier just did something. We should use something other than window.alert. I'll file a follow up for this too.